cardano-testnet/src/Testnet/Components/Configuration.hs

```diff
@@ -132,13 +132,14 @@ createSPOGenesisAndFiles
   :: (MonadTest m, MonadCatch m, MonadIO m, HasCallStack)
   => NumPools -- ^ The number of pools to make
```

Comment:
Not for this PR but we need to make a hard decision about what overrides we allow otherwise this function signature (and others) will become unwieldly. I'd say lets just stick with what we have exposed for now and then return to this once `cardano-testnet` is in a better state.

Reply:
Yep this is going to happen :+1: